app/models/tickets_competition_result.rb

```diff
@@ -58,4 +59,41 @@ def self.create_ticket!(competition, delegate_message, submitted_delegate)
       field_value: ticket_metadata.status,
     )
   end
+
+  def merge_temporary_results
```

Comment:
I am slightly opposed (but could still be convinced otherwise) to keeping the business logic within the ticket metadata.

Tickets are for describing workflows, assigning responsibilities, tracking changes, etc. They are not made to actually perform the changes.

As a further argument: If you start this way now, it would mean that in the future we have to move ~70% of our whole codebase into tickets (everything around creating, announcing, posting competitions for example), which does not feel right.

Perhaps `lib` would be the best place for this kind of code.

Reply:
I would be willing to accept a compromise where you leave `def merge_temporary_results` in the ticket, but the method body just calls the lib code.

Reply:
Okay changed to lib. I feel this can be moved completely to controller once this is the only page where this method is used. That can probably happen once the deprecated page is removed in the future.